### PR TITLE
Support css classes duplication from the overlay elementAttr option to the overlay wrapper (T824150)

### PR DESCRIPTION
**Pull request by alexander-kotov-dx**

### CODE REVIEW

**Review by mrjohn0011 — DISMISSED** [review]

File: js/ui/overlay/ui.overlay.js [hunk 1]
```diff
@@ -1118,6 +1121,45 @@ var Overlay = Widget.inherit({
         });
     },
 
+    _renderElementAttributes: function(args) {
+        this.callBase();
+        this._moveElementClassesToWrapper(args);
+    },
+
+    _moveElementClassesToWrapper: function(options) {
```
Comment (San4es) [inline js/ui/overlay/ui.overlay.js]: _moveElementClassesToWrapper -> _copyElementClassesToWrapper/_cloneElementClassesToWrapper?

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 2]
```diff
@@ -2122,6 +2122,45 @@ testModule("container", moduleConfig, () => {
         assert.ok(!$wrapper.hasClass(OVERLAY_CLASS), "only user-defined classes added to wrapper");
     });
 
+    test("css classes from the overlay elementAttr should be duplicated to the wrapper", (assert) => {
+        const instance = $("#overlay").dxOverlay({
+            visible: true,
+            elementAttr: {
+                class: "something another"
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: -> "some-class another-class"?

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 3]
```diff
@@ -2122,6 +2122,45 @@ testModule("container", moduleConfig, () => {
...
+        const $wrapper = $(instance.$content().closest(toSelector(OVERLAY_WRAPPER_CLASS)));
+
+        assert.ok($wrapper.hasClass("something"), "class added to the wrapper");
+        assert.ok($wrapper.hasClass("another"), "another class added to wrapper");
+        assert.ok($(instance.element()).hasClass("something"), "classes does not removed from the element");
+        assert.ok($wrapper.hasClass(OVERLAY_WRAPPER_CLASS), "classes does not removed from the wrapper");
+        assert.ok(!$wrapper.hasClass(OVERLAY_CLASS), "only user-defined classes added to the wrapper");
+    });
+
+    test("changed css classes from the overlay elementAttr should be used for the wrapper", (assert) => {
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js]: -> "CSS classes changes via elementAttr should be applied to the wrapper"

**Review by San4es — CHANGES_REQUESTED** [review]

**Review by San4es — APPROVED** [review]
LGTM

**Review by ksercs — APPROVED** [review]

**Comment by San4es** [comment]
We discussed this case with @alexander-kotov-dx, @NickMitrokhin, @LexDX, and found that this change can lead to unexpected behavior, so we decided not to add it to our component. We will improve our API in the future and add something like the 'wrapperAttr' option to handle similar cases.

**Pull request closed without merging**